Client/Frontend/Browser/Tab.swift

```diff
@@ -123,7 +123,7 @@ class Tab: NSObject {
 
     /// Whether or not the desktop site was requested with the last request, reload or navigation. Note that this property needs to
     /// be managed by the web view's navigation delegate.
-    var desktopSite: Bool = false
+    var desktopSite: Bool = Preferences.General.alwaysRequestDesktopSite.value ? true : false
```

Comment:
No need for `? true : false` because the `alwaysRequestDesktopSite.value` already results in a boolean.

I think we can just do:

`var desktopSite = Preferences.General.alwaysRequestDesktopSite.value`

Reply:
Lol, bad copy